bin/wasm-node/javascript/src/worker/worker.ts

```diff
@@ -57,9 +55,9 @@ const injectMessage = (instance, message) => {
     // Write the potential relay chains into memory.
     const potentialRelayChainsLen = message.potentialRelayChains.length;
     const potentialRelayChainsPtr = instance.exports.alloc(potentialRelayChainsLen * 4) >>> 0;
-    for (let idx in message.potentialRelayChains) {
+    for (let idx = 0; idx < message.potentialRelayChains.length; ++idx) {
       Buffer.from(instance.exports.memory.buffer)
-        .writeUInt32LE(message.potentialRelayChains[idx], potentialRelayChainsPtr + idx * 4);
+        .writeUInt32LE(message.potentialRelayChains[idx] as number, potentialRelayChainsPtr + idx * 4);
```

Comment:
Using `forEach` i s a bit more declarative and then there is no need to deal with possibly undefined values:

```ts
message.potentialRelayChains.forEach((potentialRelayChain, idx) => {       
  Buffer.from(instance.exports.memory.buffer)                              
    .writeUInt32LE(potentialRelayChain, potentialRelayChainsPtr + idx * 4);
})                                                                         
```

Alternatively: `message.pontentialRelayChains[idx]!` is less verbose and clearer IMO.